container_workflow_tool/utility.py

```diff
@@ -63,6 +63,11 @@ def _get_hostname_url(config):
     git_url = getattr(config, "hostname_url", None)
     return git_url
 
+def _split_config_path(config: str) -> (str, str):
```

Comment:
This can be w/o `_` at the beginning.

Can you please add into https://github.com/sclorg/container-workflow-tool/blob/master/tests/test_cli.py a test for this function and maybe also.

Reply:
I would rather keep the `_` - this function is not expected to be used by someone other than cwt itself.

Is test_cli the correct place for such a test? We should probably make test_utility if we want to test the utility functions

Reply:
> Is test_cli the correct place for such a test? We should probably make test_utility if we want to test the utility functions

OK, let's create `test_utility.py`.